src/mme-app/utils/mmeCommonUtils.cpp

```diff
@@ -335,7 +335,27 @@ ControlBlock* MmeCommonUtils::findControlBlockForS11Msg(cmn::utils::MsgBuffer* m
 
                 // Respond  with DDN failure from default DDN event handler
                 cb_p = SubsDataGroupManager::Instance()->allocateCB();
-                cb_p->setTempDataBlock(DefaultMmeProcedureCtxt::Instance());
+                cb_p->addTempDataBlock(DefaultMmeProcedureCtxt::Instance());
+            }
+        }break;
+	case create_bearer_request:
+        {
+	    const struct cb_req_Q_msg * cbr = (const struct cb_req_Q_msg *) (msg_p->getDataPointer());
+            if (cbr->s11_mme_cp_teid == 0)
```

Comment:
we already found cb based on ue index... do we need to recheck here ? i mean if s11_mme_cp_teid is 0 then mme-app would not even find cb right ?

Reply:
At this instance, we haven't found the CB. Only at this point, we are checking s11_mme_cp_teid  and if it is 0, CB will be NULL.